src/librbd/cache/ParentWriteLog.h

```diff
@@ -0,0 +1,314 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
Nit: move to the new `write_log` or `pwl` sub-directory

Reply:
Done